sql/moz-fx-data-shared-prod/braze_derived/users_v1/schema.yaml

```diff
@@ -0,0 +1,31 @@
+fields:
+- mode: NULLABLE
```

Comment:
I'm not an expert on bqetl, but should some of these fields not be nullable?

Reply:
yes! but bqetl makes all of them nullable .. even if it's the key ... 
(usually this file is autogenerated .. in this case I wrote it but I still followed the usual patterns)

Reply:
I'm no expert either...and think you are correct, just haven't seen them marked as required in other tables. @lelilia thoughts?